RELEASES.md

```diff
@@ -0,0 +1,15 @@
+# Ubuntu Advantage Client Release Process
+
+Below is the procedure used to release ubuntu-advantage-client:
+
```

Comment:
You may also want to define your release number versioning scheme.

Looking at the publishing history for ubuntu-advantage-tools, https://launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+publishinghistory, you probably want to define three different styles of versions, with some text something like this:

1.  Regular releases.  These use a "XX.Y" version number, or "XX.Y.Z" for bug fix-only releases.  So examples are "20.3" and "19.5.1".  Regular releases are uploaded to the development Ubuntu release, only.

2.  Backports.  These are copies of a regular release uploaded to the Ubuntu stable releases.  The version number for these are in the form "XX.Y~ubuntuYY.MM.N", for example "19.6~ubuntu14.04.4".

3.  Snapshots.  These are not uploaded to the Ubuntu archive, but used for build testing and/or interim distribution via PPA.  Their version numbers are in the form "XX.Y+HASHHASH~YY.MM.N", for example "20.3+gb20e0cf~20.04.1".

You don't need to use exactly what I've specified above for the version number patterns, although these conform to pretty widely adopted conventions.  But the important thing is to have a well defined standard for the project to follow, and make sure the tools adhere to it.